### ISSUE TITLE
achieved 79.11% mIoU with Resnet101 backbone

### ISSUE THREAD

**Issue by Nadavc220** [jfzhang95/pytorch-deeplab-xception#172]
First of all, thanks for creating this repo. It is very clear and easier to use than the original tensorflow model.
I ran a few training experiments on VOC2012 and saw many people trying to train the network and achieve a good result as the uploaded checkpoints (78.43%). Like many I felt there is no clear answer for those of us who uses one gpu and have a memory problem using a batch size of 16. I wanted to share the script parameters to help other people in case they found it hard to train the network above 78%.

my mIoU maxed at 79.1% around epoch 45 with the following call:
python train.py --backbone resnet --lr 0.0035 --workers 14 --epochs 50 --batch-size 8 --gpu-ids 0 --checkname deeplab-resnet --eval-interval 1 --dataset pascal

**Comment by goldhuang** [comment]
@Nadavc220 --workers 14, is it a typo?

**Comment by Nadavc220** [comment]
Weirdly enough this is not a typo.
I did not intend to run so many workers at all but I guess I did it by mistake when I ran the script.
There is no reason to use 14 workers.

**Comment by htwang14** [comment]
Did you use SBD dataset to achieve this high mIoU?

**Comment by Nadavc220** [comment]
Yes I did.
apart from the changes mentioned in the post everything was set as the
default value.

**Comment by htwang14** [comment]
Thanks!

**Comment by Alphonses** [comment]
Hi, why downloaded pretrained model can not unzip?
Can you share the pretrained model for me?

**Comment by Thunder003** [comment]
@Nadavc220 I tried to train on my custom dataset of 2 classes(1 background+1 foreground object). I have set things for 2 classes as mentioned in https://github.com/jfzhang95/pytorch-deeplab-xception/issues/117. But in the results, I'm getting a completely white image. Can you tell me what is the datatype for the image and label you used for training and what is the value assigned to label pixel corresponding to the class. Let me know what is the Pytorch version you used.

**Comment by tiesanguaixia** [comment]
> First of all, thanks for creating this repo. It is very clear and easier to use than the original tensorflow model.
> I ran a few training experiments on VOC2012 and saw many people trying to train the network and achieve a good result as the uploaded checkpoints (78.43%). Like many I felt there is no clear answer for those of us who uses one gpu and have a memory problem using a batch size of 16. I wanted to share the script parameters to help other people in case they found it hard to train the network above 78%.
> 
> my mIoU maxed at 79.1% around epoch 45 with the following call:
> python train.py --backbone resnet --lr 0.0035 --workers 14 --epochs 50 --batch-size 8 --gpu-ids 0 --checkname deeplab-resnet --eval-interval 1 --dataset pascal

excuse me, how to use the pascal 2012 dataset to train the model?
